### PR TITLE
i18n(es): update `cli-reference.md`

### PR DESCRIPTION
**Pull request by agustinmulet**
#### What kind of changes does this PR include?

- New or updated content
- Changes to the docs site code

#### Description

- Updates `cli-reference.md` in spanish
- Changes an h3 html tag to the markdown equivalent in the english version (cc @delucis @sarah11918 )

### CODE REVIEW

File: src/pages/es/reference/cli-reference.md [hunk 1]
```diff
@@ -2,63 +2,128 @@
...
 
-Establece qué direcciones IP de red debe escuchar el servidor de desarrollo (es decir, direcciones IP que no son de localhost).
+Establece qué direcciones IP de red debe escuchar el servidor de desarrollo (es decir, direcciones IP que no son de localhost). Esto puede ser útil para probar tu proyecto en dispositivos locales tales como un teléfono durante el desarrollo.
+
 - `--host` - escucha todas las direcciones, incluidas LAN y direcciones públicas
-- `--host [dirección personalizada]` - expone la dirección IP especificada en `[dirección personalizada]`
+- `--host <dirección-personalizada>` - expone la dirección IP especificada en `<dirección-personalizada>`
+
+:::caution
+No uses la flag `--host` para exponer el servidor de desarrollo en un ambiente productivo. El servidor de desarrollo está diseñado únicamente para uso local mientras desarrollas tu proyecto.
```
Comment (fredoist) [inline src/pages/es/reference/cli-reference.md:77]:
```suggestion
No uses la flag `--host` para exponer el servidor de desarrollo en un entorno de producción. El servidor de desarrollo está diseñado únicamente para uso local mientras desarrollas tu proyecto.
```

**Review by fredoist — COMMENTED** [review]
One suggestion and LGTM

**Review by fredoist — APPROVED** [review]
LGTM ✅

**Comment by kevinzunigacuellar** [comment]
Thank you @agustinmulet and @fredoist 🎉

**Pull request merged**